tests/views/test_response_correlation.py

```diff
@@ -118,16 +116,16 @@ def test_show_respo_with_obs(mock_data, dash_duo):
 
     select_ensemble(dash_duo, plugin, "default3")
 
-    resp_selector_id = plugin.uuid("parameter-selector-multi-resp")
-    default_responses = get_options(dash_duo, "#" + resp_selector_id)
-
     expected_def_responses = ["FGPT", "FOPR", "SNAKE_OIL_GPR_DIFF", "WOPR:OP1"]
-    assert default_responses == expected_def_responses
-
     expected_obs_responses = ["FOPR", "WOPR:OP1"]
+
+    resp_selector_id = "#" + plugin.uuid("parameter-selector-multi-resp")
+
+    dash_duo.wait_for_text_to_equal(resp_selector_id, "\n".join(expected_def_responses))
+
     obs_radio_btn = dash_duo.find_element(
         "#" + plugin.uuid("response-observations-check")
     )
     obs_radio_btn.click()
-    obs_responses = get_options(dash_duo, "#" + resp_selector_id)
-    assert obs_responses == expected_obs_responses
+
+    dash_duo.wait_for_text_to_equal(resp_selector_id, "\n".join(expected_obs_responses))
```

Comment:
Any reason to not having the timeout specification? `, timeout=5`

Reply:
i like going with sane defaults, assuming that dash put a sane default : )

but if you think it's too much (is it 10 seconds, i don't remember), we can put our own value, but we probably have to do it all over... i dunno

Reply:
Maybe my question was misleadingly in the wrong place...I noticed that on other parts of the code a timeout was specified as 5 (e.g. lines 86 and 90), but not here. If there is no argument to add it here, is it an argument to remove the timeout of the others? 🤔

Reply:
made issue #362